pkgs/by-name/ca/cacti/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "cacti";
+  version = "7.0";
+
+  src = fetchFromGitHub {
+    owner = "HewlettPackard";
+    repo = "cacti";
+    rev = "1ffd8dfb10303d306ecd8d215320aea07651e878";
+    sha256 = "sha256-lrbrwKlaVvwEUDZA/n8I/zYNX3T8ltiBTYL94Ce5UQU=";
```

```suggestion
    hash = "sha256-lrbrwKlaVvwEUDZA/n8I/zYNX3T8ltiBTYL94Ce5UQU=";
```
for sri hash use hash